assets/js/components/legacy-setup/search-console.js

```diff
@@ -74,7 +74,7 @@ class SearchConsole extends Component {
 		const { setErrorMessage } = this.props;
 		( async () => {
 			try {
-				const properties = await data.get( TYPE_MODULES, 'search-console', 'matched-sites' );
+				const properties = await API.get( 'modules', 'search-console', 'matched-sites' );
```

Comment:
Here's another example which looks harmless but remember that the default value of the parameter controlling whether to use cache or not is not the same between the two.

The legacy dataAPI defaults to bypassing cache, while the modern API defaults to using cache **(only relevant for `.get`)**
https://github.com/google/site-kit-wp/blob/3db19cbeca41145aefc6c6c3383ee9b66e8b4121/assets/js/components/data/index.js#L240

So here we'd need to set `useCache: false` to be consistent, which is something we generally do for all requests made during set up.